arangod/Cluster/MaintenanceRestHandler.cpp

```diff
@@ -121,6 +121,7 @@ bool MaintenanceRestHandler::parsePutBody(VPackSlice const& parameters) {
 
   std::map<std::string, std::string> desc;
   auto prop = std::make_shared<VPackBuilder>();
+  int priority = 1;
```

Comment:
Do you really want the default to be "top of the priority list"?

Reply:
`priority_queue` works the other way round, high numbers are high priority, so this is by default the lowest. That was at least my thinking.